standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -1028,11 +1028,11 @@ public boolean checkFailedCompactions(CompactionInfo ci) throws MetaException {
         return (numFailed == failedThreshold) && !needsRetry;
       }
       catch (SQLException e) {
-        LOG.error("Unable to check for failed compactions " + e.getMessage());
+        LOG.error("Unable to check for failed compactions" + e.getMessage());
```

Comment:
I don't see any reason not to log out the full stack trace even if we have to retry it later, but if we decide to log only the error message, then we need that space, or even better a ": ".

Reply:
I very much agree, but was trying to keep the number of changes down.  I will have it print stack trace.

Reply:
@miklosgergely Change made.  Please review. :)